src/renderer/tvgLoader.cpp

```diff
@@ -368,7 +368,7 @@ LoadModule* LoaderMgr::loader(const char* data, uint32_t size, const char* mimeT
                 }
                 return loader;
             } else {
-                TVGLOG("LOADER", "Given mimetype \"%s\" seems incorrect or not supported.", mimeType);
+                TVGLOG("LOADER", "Given mimetype seems incorrect or not supported.");
```

Comment:
At 361 line should be `mimeType` instead of `!mimeType`. Seems a wrong logic.

Reply:
Done it :^)

Reply:
@faxe1008 awesome, thanks!